_data/sidebars/home_sidebar.yml

```diff
@@ -74,6 +74,8 @@
           url: /scheduler/kubernetes/failover.html
         - title: Run Hyper Converged
           url: /scheduler/kubernetes/scheduler-convergence.html
+        - title: Update Portworx Rack Info
```

Comment:
Maybe we should rename this? This talks about Rack, Zone (and Region if we add that). Should we rename it to "Update Portworx Geography (or Locality) Info"